core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java

```diff
@@ -85,12 +86,17 @@ public boolean isDisableChecks() {
         return Boolean.parseBoolean((String) environmentProperties.getOrDefault("checks.disable", "false"));
     }
 
+    @UnstableAPI
+    public boolean environmentSupportsReuse() {
+        return Boolean.parseBoolean((String) environmentProperties.getOrDefault("testcontainers.reuse.enable", "false"));
```

Comment:
Is there a reason this is fetched from the `environmentProperties` and not just the general properties? I just added this property to my `testcontainers.properties` file on my classpath and was wondering for a long time why this wasn't working until I saw this line.

Reply:
See this thread:
https://github.com/testcontainers/testcontainers-java/pull/1781#discussion_r318675046

TL;DR: it is a parameter of every environment and cannot be shared / committed to the repo. Also, it **SHOULD NOT** be used on CI environments.
The PR's description states it pretty clearly and does not mention the classpath file anywhere :)